### ISSUE TITLE
(7.2) Consistent name ordering: LAST, First not in Member Roster

### ISSUE THREAD

**Issue by drodruck** [josephr5000/confdb-public#453]
RELATED TO: #448

ENVIRONMENT
Windows 10 Pro, Firefox and Chrome

ISSUE
(7.2) Consistent name ordering: LAST, First not in Member Roster

WHAT PAGE
Members/Officers directory

WHAT STEPS WHERE TAKEN
Clicked “Members/Officers” from the Main page

WHAT I EXPECTED
The list of members would be displayed last name first.

WHAT HAPPENED
The list displayed first name first.

Screen Shots
![image](https://user-images.githubusercontent.com/20655031/35716468-fbf1f1e2-078d-11e8-9b90-06f90877ecda.png)

**Comment by drodruck** [comment]
Also in Outstanding Pledges, and Pledge Totals Report.

**Comment by josephr5000** [comment]
Re members, at least we're consistent.  The add/edit member page is also in first/last order.  My assumption was that the key goal was to be sure all *client* info was in last/first order.  We could switch members also, but last/first always feels so formal, and I wonder if we want members listed that formally?  Perhaps we do.  Meeting discussion topic, perhaps?

I'll check out Outstanding Pledges and Pledge Totals.

**Comment by josephr5000** [comment]
**FIXED** Outstanding Pledges and Pledge Totals.  I'm going to tag this fixed, understanding that you may bring the member question to the next meeting, okay?  Thanks.

**Comment by drodruck** [comment]
Agreed, Outstanding Pledges and Pledge Total fixed and the member question is something for the Committee.
